src/Workspaces/CSharp/Portable/Simplification/Reducers/CSharpMiscellaneousReducer.cs

```diff
@@ -112,7 +113,7 @@ public CSharpMiscellaneousReducer() : base(s_pool)
         {
             if (node.Statements.Count == 1 &&
                 CanHaveEmbeddedStatement(node.Parent) &&
-                !optionSet.GetOption(CSharpCodeStyleOptions.PreferBraces).Value)
+                optionSet.GetOption(CSharpCodeStyleOptions.PreferBraces).Value == PreferBracesPreference.None)
```

Comment:
📝 I expect this to cause a test failure in `AddParameterCheckTests.TestPreferNoBlock`. I'll revisit this tomorrow to get the pair aligned.

Reply:
➡️ Fixed in cc5f35bdedef1e27066949defcd31602adc1212f